test/appium/tests/critical/test_public_chat_browsing.py

```diff
@@ -351,6 +352,56 @@ def test_community_copy_and_paste_message_in_chat_input(self):
 
         self.errors.verify_no_errors()
 
+    @marks.testrail_id(703133)
+    def test_restore_multiaccount_with_waku_backup_remove_switch(self):
+        self.home.click_system_back_button_until_element_is_shown()
+        profile = self.home.profile_button.click()
+        profile.logout()
+        self.sign_in.recover_access(passphrase=waku_user.seed, second_user=True)
+
+        self.home.just_fyi("Restore user with predefined communities, check communities")
+        self.home.communities_tab.click()
+        for key in ['admin_open', 'member_open', 'admin_closed', 'member_closed']:
+            if not self.home.element_by_text(waku_user.communities[key]).is_element_displayed(30):
+                self.errors.append("%s was not restored from waku-backup!!" % key)
+        self.home.opened_communities_tab.click()
+        if not self.home.element_by_text(waku_user.communities['member_pending']).is_element_displayed(30):
+            self.errors.append("Pending community %s was not restored from waku-backup!" % waku_user.communities['member_pending'])
```

Comment:
Why do we want to see Pending community on Opened tab?

Reply:
because the membership is expired and you need to re-request membership in order to move it to 'Pending'